osp-deploy/cinder/powerflex/README.md

```diff
@@ -152,3 +156,70 @@ Install the Storage Data Client (SDC) on all nodes after deploying the overcloud
 ### Testing the configured Backend
 Finally create a volume-type and test if you can successfully create and attach volumes of that type.
 
+## Upgrading from RHOSP13 to RHOSP16.1
+
+### Upgrading PowerFlex
+
+At the first step, upgrade PowerFlex system to the latest available version using official instructions. 
+Upgrading SDC is not necessary as it has to be upgraded after Overcloud nodes OS upgrade.
+For more information on how to upgrade PowerFlex system please contact [PowerFlex support team](https://www.dell.com/support/contents/en-us/category/contact-information).
+
+### Upgrading RHOSP
+
+For instructions of how to upgrade RHOSP13 to RHOSP16.1 please refer to [In-place upgrades from Red Hat OpenStack Platform 13 to 16.1](https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/framework_for_upgrades_13_to_16.1).
+
+### Additional steps that have to be done to get working environment after upgrade
+
+If you use the environment file with configuration from previous installation (RHOSP13) some changes have to be done:
+
+* No custom containers for cinder/nova/glance services are needed in RHOSP16.1. They must be removed from environment file.
+  However, custom volume mappings are still necessary.
+
+* **cinder.volume.drivers.dell_emc.scaleio.driver.ScaleIODriver** was deprecated and removed. sio_* options have to be renamed.
+  Do not change **volume_backend_name** as it is used in Cinder Volume Type object.
+
+This file has to be included in openstack `overcloud upgrade prepare` and `openstack overcloud upgrade converge` commands.
```

Comment:
What file is that ? 
Are you referring to configuration file who sample is below?
Is that cinder.conf file?

Reply:
Yep, the sample below is an example with diff changes (red-green). You can see compiled file looks in branch
(https://github.com/dell/osp-integration/tree/rhosp13-16_upgrade_instructions/osp-deploy/cinder/powerflex).

Reply:
Since that is what we had done in previous upgrade we are good. 
While I still do not like "This File" I am ok with it.